queries.sql

```diff
@@ -82,3 +82,178 @@ SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+  1 | Agumon     | 2020-02-03    |               0 | t       |     10.23 | digimon
+  2 | Gabumon    | 2018-11-15    |               2 | t       |      8.00 | digimon
+  4 | Devimon    | 2017-05-12    |               5 | t       |     11.00 | digimon
+  6 | Plantmon   | 2021-11-15    |               2 | t       |     -5.70 | digimon
+  8 | Angemon    | 2005-06-12    |               1 | t       |    -45.00 | digimon
+  9 | Boarmon    | 2005-06-07    |               7 | t       |     20.40 | digimon
+(11 rows)
+*/
+
+-- Start the transaction
+START TRANSACTION;
+
+-- Delete all records from the animals table
+DELETE FROM animals;
+
+-- Roll back the transaction to undo the delete
+ROLLBACK;
+
+-- Start the transaction
+START TRANSACTION;
+
+-- Delete all animals born after Jan 1st, 2022
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+-- Create a savepoint for the transaction
+SAVEPOINT weight_update;
+
+-- Update all animals' weight to be their weight multiplied by -1
+UPDATE animals SET weight_kg = weight_kg * -1;
+
+-- Rollback to the savepoint
+ROLLBACK TO weight_update;
+
+-- Update all animals' weights that are negative to be their weight multiplied by -1
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+
+-- Commit the transaction
+COMMIT;
+/* 
+ id |    name    | date_of_birth | escape_attempts | neutred | weight_kg |   species
+----+------------+---------------+-----------------+---------+-----------+--------------
+ 10 | Blossom    | 1998-10-13    |               3 | t       |     17.00 | Flower
+  3 | Pikachu    | 2021-01-07    |               1 | f       |     15.04 | pokemon
+  1 | Agumon     | 2020-02-03    |               0 | t       |     10.23 | digimon
+  2 | Gabumon    | 2018-11-15    |               2 | t       |      8.00 | digimon
+  4 | Devimon    | 2017-05-12    |               5 | t       |     11.00 | digimon
+  9 | Boarmon    | 2005-06-07    |               7 | t       |     20.40 | digimon
+  5 | Charmander | 2020-02-08    |               0 | f       |     11.00 | Fire Lizard
+  7 | Squirtle   | 1993-04-02    |               3 | f       |     12.13 | Water Turtle
+  6 | Plantmon   | 2021-11-15    |               2 | t       |      5.70 | digimon
+  8 | Angemon    | 2005-06-12    |               1 | t       |     45.00 | digimon
+(10 rows)
+*/
+
+-- How many animals are there?
+SELECT COUNT(*) FROM animals;
+/* 
+ count
+-------
+    10
+(1 row)
+*/
+-- How many animals have never tried to escape?
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+/* 
+ count
+-------
+     2
+(1 row)
+*/
+
+-- What is the average weight of animals?
+SELECT AVG(weight_kg) FROM animals;
+/* 
+         avg
+---------------------
+ 15.5500000000000000
+(1 row)
+*/
+
+-- Who escapes the most, neutered or not neutered animals?
+SELECT neutred, COUNT(*) FROM animals GROUP BY neutred;
+/*
+ neutred | count
+---------+-------
+ f       |     3
+ t       |     7
+(2 rows)
+*/
```

Comment:
Kindly update this statement to show the total number of escape attempts of neutered animals vs the total number of escape attempts of not neutered animals instead of simply counting how many neutered and non neutered animals there are🙏